src/DoctrineWriter.php

```diff
@@ -262,47 +287,50 @@ protected function disableLogging()
      */
     protected function reEnableLogging()
     {
-        $config = $this->entityManager->getConnection()->getConfiguration();
+        //TODO: do we need to add support for MongoDB logging?
+        if (!($this->objectManager instanceof \Doctrine\ORM\EntityManager)) return;
```

Comment:
Ah, so this is an internal thing. Still, a separated class would simply solve this issue.